data/remnant/remnant outfits.txt

```diff
@@ -478,6 +479,10 @@ outfit "Anvil-Class Engine"
 	"turn" 336.0
 	"turning energy" 1.05
 	"turning heat" 1.1
+	"reverse thrust" 8.1
+	"reverse thrusting energy" 1.23
+	"reverse thrusting heat" 1.17
```

Comment:
Even without adding the flare points on Remnant ships, you still need a reverse flare sprite and reverse flare sound.
```suggestion
	"reverse thrust" 8.1
	"reverse thrusting energy" 1.23
	"reverse thrusting heat" 1.17
```

Reply:
Alright, I'll do that next.

Reply:
Done. Now to test.